README.md

```diff
@@ -9,6 +9,8 @@
 ## Setup
 
 * Clone the repo
+* Download JDK8 from [here](https://www.oracle.com/in/java/technologies/javase/javase-jdk8-downloads.html)
+* Update your JDK location via Project Structure (Android Studio)
```

Comment:
Can we add the steps to update JDK location on android studio?

Reply:
added